redbot/core/core_commands.py

```diff
@@ -112,7 +112,7 @@ def __init__(self, bot: "Red"):
         self.bot.register_rpc_handler(self._name)
         self.bot.register_rpc_handler(self._prefixes)
         self.bot.register_rpc_handler(self._version_info)
-        self.bot.register_rpc_handler(self._invite_url)
+        self.bot.register_rpc_handler(bot.invite_url)
```

Comment:
This needs to stay as it was and the `_invite_url` method here should just call the bot's `get_invite_url()` method. Otherwise, the RPC handler won't get auto-unregistered (well, it will because it's Core and it never gets unloaded but still, I would like this to be done properly).